Jasmine/Common/Models/LevelResult.swift

```diff
@@ -0,0 +1,46 @@
+import RealmSwift
+
+/// Simple container for information necessary at the end of every game
+class LevelResult: Object {
+
+    private static let rawLostInt = 0
```

Comment:
@donjar GameStatus doesn't have a rawValue since it's not a primitive, should I change it?